crates/sargon/src/profile/v100/entity/persona/persona_data/persona_data_identified_collection_types.rs

```diff
@@ -28,7 +28,6 @@ macro_rules! declare_collection_of_identified_entry {
             Eq,
             derive_more::Display,
             derive_more::Debug,
-            uniffi::Record,
```

Comment:
We do not need to do it now, but this whole macro should be removed. This is one of many macros which can now be replaced with generics thx to not being UniFFI compatible anymore.